package com.kh.chap04_userException.controller;

import java.util.Scanner;

public class UserExceptionController {
	// 숫자 3개를 입력받아 합을 출력하는 메소드
	// 단, 0 입력 시 바로 프로그램 종료
	
	public void method1() {
		Scanner sc = new Scanner(System.in);
		int a = 0;
		int sum = 0;
		for (int i = 0; i < 3; i++) {
			System.out.print(i+1+"번째 숫자 입력 : ");
			a = sc.nextInt();
			if(a==0) {
				System.out.println("프로그램 강제 종료");
				return;
			}
			sum += a;
		}
		System.out.println("입력하신 숫자의 합 : " + sum);
		System.out.println("프로그램 정상 종료");
		return;
	}
	
	
	public void method2() {
		Scanner sc = new Scanner (System.in);
		
		int sum = 0;
		int input = 0;
		for (int i = 1; i < 4; i++) {
			System.out.println("입력 : " + i + " : ");
			input = sc.nextInt();
			if(input == 0) {
				System.out.println("프로그램 강제 종료");
				return;
			}
			sum += input;
		}
		
		System.out.println("합 : " + sum);
		System.out.println("프로그램 정상 종료");
	}
	
	
}
